### PR TITLE
Replace "+mm" with "mm." Allow negative recover-swap length.

### PR DESCRIPTION
**Pull request by thinkyhead**
Applies to MarlinFirmware/Marlin#5550

### CODE REVIEW

File: Marlin/language_en.h [hunk 1]
```diff
@@ -364,10 +364,10 @@
   #define MSG_CONTROL_RETRACT_ZLIFT           _UxGT("Hop mm")
 #endif
 #ifndef MSG_CONTROL_RETRACT_RECOVER
-  #define MSG_CONTROL_RETRACT_RECOVER         _UxGT("UnRet +mm")
+  #define MSG_CONTROL_RETRACT_RECOVER         _UxGT("UnRet mm")
 #endif
 #ifndef MSG_CONTROL_RETRACT_RECOVER_SWAP
-  #define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("S UnRet+mm")
+  #define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("S UnRetmm")
```
Comment (esenapaj) [inline Marlin/language_en.h]: Wasn't spacing needed between "Unret" and "mm"?

File: Marlin/language_el-gr.h [hunk 2]
```diff
@@ -141,8 +141,8 @@
 #define MSG_CONTROL_RETRACT_SWAP            _UxGT("Εναλλαγή ανάσυρσης μμ")
 #define MSG_CONTROL_RETRACTF                _UxGT("Ανάσυρση V")
 #define MSG_CONTROL_RETRACT_ZLIFT           _UxGT("Μεταπήδηση μμ")
-#define MSG_CONTROL_RETRACT_RECOVER         _UxGT("UnRet +mm")
-#define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("S UnRet+mm")
+#define MSG_CONTROL_RETRACT_RECOVER         _UxGT("UnRet mm")
+#define MSG_CONTROL_RETRACT_RECOVER_SWAP    _UxGT("S UnRetmm")
```
Comment (esenapaj) [inline Marlin/language_el-gr.h]: Same as [previous comment](https://github.com/FHeilmann/Marlin/pull/8/files#r93202601).

**Comment by esenapaj** [comment]
The same also applies to cn, cz, da, hr, it, tr.

**Comment by thinkyhead** [comment]
Ah, yes! MarlinFirmware/Marlin#5565

**Pull request merged**